### PR TITLE
Documentation with tutorial

### PR DESCRIPTION
**Pull request by RafaelPignata622**
I modified the documentation, added the tutorial, the license.rst file, a new module with 3 functions that loads the files star,dark,gas_.dat that is used by the tutorial.

### CODE REVIEW

**Review by nvillagra — COMMENTED** [review]
Looking good

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,27 +1,48 @@
-.. galaxy-chop documentation master file, created by
+.. GalaxyChop documentation master file, created by
    sphinx-quickstart on Sun Oct 11 11:10:44 2020.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to galaxy-chop's documentation!
+Welcome to GalaxyGhop's documentation!
```
Comment (ornemarioni) [inline docs/source/index.rst:6]: GalaxyChop*

Comment (RafaelPignata622) [reply docs/source/index.rst:6]: sin el 's?

**Review by ornemarioni — COMMENTED** [review]
all looks good

File: docs/source/index.rst (hunk 1 above)
Comment (ornemarioni) [reply docs/source/index.rst:6]: No, the G for C

Comment (RafaelPignata622) [reply docs/source/index.rst:6]: ahhhh jajajaja okay

**Pull request merged**